build.rs

```diff
@@ -132,9 +142,9 @@ fn main() {
 	println!(
 		"cargo:rustc-link-search={}",
 		Path::new(&nrfxlib_path)
-			.join("crypto/nrf_oberon/lib/cortex-m33/hard-float")
+			.join("crypto/nrf_oberon/lib/cortex-m33/soft-float")
```

Comment:
Why soft float? We have an FPU, so why not use it?

Reply:
You resolved this, but it's still soft float...

Reply:
Sorry.  Reverted the modem library but not the oberon library.